barretenberg/cpp/src/barretenberg/honk/sumcheck/sumcheck_output.hpp

```diff
@@ -15,31 +16,7 @@ template <typename Flavor> struct SumcheckOutput {
     std::vector<FF> challenge_point;
     // Evaluations in `u` of the polynomials used in Sumcheck
     ClaimedEvaluations purported_evaluations;
-
-    SumcheckOutput()
-        : purported_evaluations(std::array<FF, Flavor::NUM_ALL_ENTITIES>()){};
-
-    SumcheckOutput(const std::vector<FF>& _challenge_point, const ClaimedEvaluations& _purported_evaluations)
-        : challenge_point(_challenge_point)
-        , purported_evaluations(_purported_evaluations){};
-
-    SumcheckOutput& operator=(SumcheckOutput&& other)
-    {
-        challenge_point = other.challenge_point;
-        purported_evaluations = other.purported_evaluations;
-        return *this;
-    };
-
-    SumcheckOutput(const SumcheckOutput& other)
-        : challenge_point(other.challenge_point)
-        , purported_evaluations(other.purported_evaluations){};
-
-    bool operator==(const SumcheckOutput& other) const
-    {
-        bool result{ false };
-        result = challenge_point == other.challenge_point;
-        result = purported_evaluations._data == other.purported_evaluations._data;
-        return result;
-    };
+    // Whether or not the purported multilinear evaluations and final sumcheck evaluation have been confirmed
```

Comment:
can we have claimed/alleged evaluations rather than purported, I am a big fan of simple words in comments (:

Reply:
changed the member and the comment to use "claimed"